test/test_suite.bats

```diff
@@ -179,54 +179,54 @@
 @test "Get all queries" {
   run bash -c 'echo ">getallqueries >quit" | nc -v 127.0.0.1 4711'
   printf "%s\n" "${lines[@]}"
-  [[ ${lines[1]} == *"TXT version.ftl 127.0.0.1 3 0 6"* ]]
-  [[ ${lines[2]} == *"TXT version.bind 127.0.0.1 3 0 6"* ]]
-  [[ ${lines[3]} == *"A blacklisted.com 127.0.0.1 5 0 4"* ]]
-  [[ ${lines[4]} == *"A 0427d7.se 127.0.0.1 1 0 4"* ]]
-  [[ ${lines[5]} == *"A whitelisted.com 127.0.0.1 2 0 4"* ]]
-  [[ ${lines[6]} == *"A regex5.com 127.0.0.1 4 0 4"* ]]
-  [[ ${lines[7]} == *"A regexa.com 127.0.0.1 2 0 7"* ]]
-  [[ ${lines[8]} == *"A google.com 127.0.0.1 2 0 4"* ]]
-  [[ ${lines[9]} == *"AAAA google.com 127.0.0.1 2 0 4"* ]]
-  [[ ${lines[10]} == *"A ftl.pi-hole.net 127.0.0.1 2 0 4"* ]]
+  [[ ${lines[1]} == *"TXT version.ftl 127.0.0.1 "*"3 0 6"* ]]
```

Comment:
Here and below: Acknowledge that there might be a host name after the IP address of the client.